app/presenters/facility_management.rb

```diff
@@ -0,0 +1,14 @@
+class FacilityManagement
+
+  def self.register_payment_type(type, options = {})
+    options[:short_name] ||= type.name.underscore
+    payment_types[type] = options
+  end
+
+  private
+
+  def self.payment_types
+    @@payment_types ||= {}
```

Comment:
Will we ever care about having control over the order of payment types?  I assume, no.